### ISSUE TITLE
outdated documentation?

### ISSUE THREAD

**Issue by nyxnor** [gkdr/lurch#174]
https://github.com/gkdr/lurch#caveats
>For instance, if you don't install the additional plugin mentioned above, this is probably not the right thing to use if you have multiple clients running at the same time, as there is no message carbons support in libpurple as of now.

But there is this https://github.com/gkdr/lurch#carbons
From your other repo https://github.com/gkdr/carbons

**Comment by gkdr** [comment]
You're right, and this is what I meant by "the additional plugin mentioned above". I admit that it could be a lot clearer :slightly_smiling_face: (and to be honest, the whole README needs a redo.)

I think I just wanted to emphasize the sentence before:
>OMEMO is not 'whatever Conversations can do', but a very specific XEP.
